cmake/Dependencies.cmake

```diff
@@ -48,7 +48,7 @@ list(APPEND Peloton_LINKER_LIBS ${PQXX_LIBRARIES})
 list(APPEND Peloton_LINKER_LIBS "-lssl")
 
 # --[ LLVM 3.7+
-find_package(LLVM 3.7 REQUIRED CONFIG)
+find_package(LLVM REQUIRED CONFIG)
```

Comment:
What is a better version comparison to use?  With just `LLVM 3.7 REQUIRED CONFIG`, I got build failures on LLVM 3.9+ systems.